docs/overview.md

```diff
@@ -1,44 +1,41 @@
-  edited rapidly and easily.
-- Journals can be stored anywhere, including in shared folders so that they can
-  be synchronized between devices.
-- Journal entries can be created and modified using an external editor of your
-  choice, using the command line as a starting point to filter specific entries.
-- BeJournals are stored in single files using date/time tags to identify
-  individual entries. Some systems endorse a file/folder methodology; `jrnl`
-  keeps 
-- Journals can be protected using [256-bit
-  AES encryption](http://en.wikipedia.org/wiki/Advanced_Encryption_Standard).
-
-## Why keep a journal?
-
-Journals aren't just for people who have too much time on their summer vacation.
-Journals help you keep track of events, thoughts, feelings, tasks, ideas, and
-anything else that is worth remembering. Your imagination may be limitless, but
-your memory probably isn't.
-
-To begin, try to write at least 20 words a day. `jrnl` makes this incredibly
-easy. Fire up a terminal and type, `jrnl today at 8am: I arrived at work to find
-a birthday present on my desk. My colleagues are the best!` You can reflect on
-what made your day special so that you'll always remember why you haven't wasted
-it. Or try using `jrnl` to keep a gratitude journal, a habit that has been shown
-to improve mental health and quality of life.
-
-For professional use, a text-based journal can be the perfect complement to your
-to-do list -- a record of what you've done as well as how and when you've done
-it. You can use a journal as a quick way to keep a change log (`jrnl Updated
-jrnl overview page to make it more awesomer`). Or use it to keep a lab notebook.
-The possibilities are almost endless.
+Your imagination may be limitless, but your memory probably isn't. `jrnl` lets
+you write something down, and then it gets out of the way.
+
+## how it works
+
+New entries are created on the command line:
+
+``` sh
+jrnl today at 8am: I arrived at work to find a birthday present on my desk. My colleagues are the best!
+```
+
+Support for external editors is included.
+
+### one file type to rule them all...
+
+`jrnl` stores your journals as human-readable, future-proof plain text files.
+You can store them wherever you want, including in shared folders to keep them
+synchronized between devices. And because journal files are stored as plain
+text, you can rest assured that your journals will be readable for centuries.
+
+### ...and as many files as you need
+  
+`jrnl` allows you to work with multiple journals, each of which is stored as a
+single file using date and time tags to identify individual entries. `jrnl`
+makes it easy to find the entries you want, and only the ones you want, so that
+you can read them or edit them. Here's an example: say you want to find all of
+the entries you wrote on January 3rd, 2020 that include the word _cat_, and
+change every instance of the word _cat_ to _dog_? Easy:
```

Comment:
This doesn't actually change anything, it ONLY gets entries with _cat_, so the text above it is inaccurate.

Reply:
Right, I should clarify, you still have to do the find-and-replace-all in the editor, but at least it only shows the ones that matter. I'll think of a better example.

Reply:
I think removing the part about `_dog_` here would be enough to clarify the example.